external/fv3fit/fv3fit/_shared/scaler.py

```diff
@@ -226,29 +127,3 @@ def loads(b: str) -> NormalizeTransform:
             return scaler_cls.load(f)
 
     raise NotImplementedError(f"Cannot load {class_name} scaler")
-
-
-def get_scaler(
```

Comment:
I don't see any changes to the fv3fit RF module in this PR, was it changed previously to remove its usage of this function?